about.css

```diff
@@ -19,18 +19,36 @@ section div {
     width: 50%;
 }
 
+/* commitment to authenticity */
+.commitment {
+    border-radius: 30px;
+}
+
+.commitment .text-content {
+    padding: 30px;
+}
+
+
+
 
 /* TEAM MEMBERS */
 .team-container {
-    width: 90%;
+    width: 95%;
```

Comment:
We might need a max-width on this, I am not sure.

Reply:
added a max-width of 1600px